### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by alberttwong**
Added text to say that Atlas based organizations in cloud.mongodb.com are not supported.

https://github.com/mongodb/mongodb-enterprise-kubernetes/issues/155

### CODE REVIEW

**Comment by rodrigovalin** [comment]
@alberttwong Can you please sign our CLA so we can merge your patch?

Here's a link: https://www.mongodb.com/legal/contributor-agreement

**Comment by rodrigovalin** [comment]
@alberttwong I think there's a mistake in here. There's no "Ops Manager" based organization in Cloud Manager; Ops and Cloud Manager are two different products.

**Comment by alberttwong** [comment]
I didn't say cloud manager.  I said cloud.mongodb.com.   Here's the screenshot that you need to make the correct choice.   

Screen Shot 2020-11-11 at 8 00 56 AM

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Welcome to the MongoDB Enterprise Kubernetes Operator. The Operator enables easy
 * Ops Manager - our enterprise management, monitoring and backup platform for MongoDB. The Operator can install and manage Ops Manager in Kubernetes for you.
 
 The Operator requires access to one of our database management tools - Ops Manager or Cloud Manager - to deploy MongoDB instances. 
-You may run Ops Manager either inside or outside Kubernetes, or may use Cloud Manager (cloud.mongodb.com) instead.
+You may run Ops Manager either inside or outside Kubernetes, or may use Cloud Manager (Ops Manager-based organization on cloud.mongodb.com, Atlas-based organization will not work) instead.
```
Comment (rodrigovalin) [inline README.md:8]:
```suggestion
You may run Ops Manager either inside or outside Kubernetes, or may use Cloud Manager (Cloud Manager-based organization on cloud.mongodb.com, Atlas-based organization will not work) instead.
```

File: README.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ to see which Kubernetes and Openshift versions the Operator is compatible with
 To work with MongoDB resource this Operator requires [Ops Manager](https://docs.opsmanager.mongodb.com/current/) (Ops Manager can 
 be installed into the same Kubernetes cluster by the Operator or installed outside of the cluster manually) 
 or [Cloud Manager](https://cloud.mongodb.com/user#/cloud/login).
-> If this is your first time trying the Operator, Cloud Manager is easier to get started
+> If this is your first time trying the Operator, Cloud Manager is easier to get started.  Use or create a Ops Manager based organization.  Atlas-based organizations will not work.
```
Comment (rodrigovalin) [inline README.md:39]:
```suggestion
> If this is your first time trying the Operator, Cloud Manager is easier to get started.  Use or create a Cloud Manager based organization.  Atlas-based organizations will not work. 
```

**Comment by alberttwong** [comment]
thanks for catching my mistake.

File: README.md (hunk 1 above)
Comment (rodrigovalin) [reply README.md:8]: This one also needs to be changed.

Comment (jamesbroadhead) [reply README.md:8]: I don't think that we should have this caveat in the first 10 lines of our readme - can we stick with the reference lower down instead?

File: README.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ to see which Kubernetes and Openshift versions the Operator is compatible with
 To work with MongoDB resource this Operator requires [Ops Manager](https://docs.opsmanager.mongodb.com/current/) (Ops Manager can 
 be installed into the same Kubernetes cluster by the Operator or installed outside of the cluster manually) 
 or [Cloud Manager](https://cloud.mongodb.com/user#/cloud/login).
-> If this is your first time trying the Operator, Cloud Manager is easier to get started
+> If this is your first time trying the Operator, Cloud Manager is easier to get started.  Use or create a Cloud Manager based organization.  Atlas-based organizations will not work.
```
Comment (jamesbroadhead) [inline README.md:39]: our official terminology is "Cloud Manager Organization / Atlas Organization" - could you drop the "-based" suffix here please?

**Comment by jamesbroadhead** [comment]
We're merging these changes into the readme - it will be updated on our next push to this repo.

**Pull request closed without merging**